Makefile

```diff
@@ -52,8 +52,19 @@ fmt:
 
 __REWARDS__: ##
 
-create-and-claim-distribution:
-	./contracts/anvil/create-and-claim-distribution.sh
-
-create-operator-directed-distribution-and-claim:
-	./contracts/anvil/create-operator-directed-and-claim-distribution.sh
+SENDER_ADDR=0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
+
+create-avs-distributions-root:
+	cd contracts && \
+	forge script script/SetupDistributions.s.sol:SetupDistributions --rpc-url http://localhost:8545 \
+		--broadcast --sig "runAVSRewards()" -v --sender "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
+
+create-operator-directed-distributions-root:
+	cd contracts && \
+	forge script script/SetupDistributions.s.sol:SetupDistributions --rpc-url http://localhost:8545 \
+		--broadcast --sig "runOperatorDirected()" -v --sender "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
```

Comment:
Let's use variables for `"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"` and the anvil address